lessons/13-server-rendering/README.md

```diff
@@ -27,21 +27,17 @@ var path = require('path')
 
 module.exports = {
 
-  entry: path.resolve(__dirname, 'server.js'),
+  entry: './server.js',
```

Comment:
This will cause issues on certain environments. `path.resolve` is the safest option here.